File: sdkwork-ai-sdk-csharp/Models/ImageGenerationRequest.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class ImageGenerationRequest
    {
        public string? Model { get; set; }
        public string? Prompt { get; set; }
        public int? N { get; set; }
        public string? Quality { get; set; }
        public string? Size { get; set; }
        public string? Style { get; set; }
        public string? User { get; set; }
        public List<string>? Image { get; set; }
        public Dictionary<string, object>? Extra { get; set; }
        public string? ResponseFormat { get; set; }
    }
}

File: sdkwork-ai-sdk-go/types/image_generation_request.go
package types

// Image generation request
type ImageGenerationRequest struct {
	Model string `json:"model"`
	Prompt string `json:"prompt"`
	N int `json:"n"`
	Quality string `json:"quality"`
	Size string `json:"size"`
	Style string `json:"style"`
	User string `json:"user"`
	Image []string `json:"image"`
	Extra map[string]interface{} `json:"extra"`
	ResponseFormat string `json:"response_format"`
}
